frontend/typedefs/common.js

```diff
@@ -0,0 +1,3 @@
+/**
```

Comment:
What is this? Do we need it? AFAIK vscode and web/phpstorm knows the JSX type.

Also the same definition exists in `frontend/typedefs/jsx.js`